lua/efmls-configs/linters/jsonlint.lua

```diff
@@ -0,0 +1,19 @@
+-- Metadata
+-- languages: JSON
+-- url: https://github.com/zaach/jsonlint
+
+local sourceText = require('efmls-configs.utils').sourceText
+local fs = require('efmls-configs.fs')
+
+local linter = 'jsonlint'
+local args = '--sort-keys  --compact --validate ${INPUT}'
+local command = string.format('%s %s', fs.executable(linter, fs.Scope.NODE), args)
+
+return {
+	prefix = linter,
+	lintSource = sourceText(linter),
+	lintCommand = command,
+	lintStdin = true,
+	lintFormats = { '%.%#: line %l, col %c, %trror - %m', '%.%#: line %l, col %c, %tarning - %m' },
```

Comment:
I'm not getting any error/warming messages with the provided `lintFormats`

Reply:
it work before big update on efm and alot of other main plugin(treesitter, lspconf, lazy, mason) and last neovim nighty . i last us it on efm ver.0.48. after that. efm stop working in all config not just lintformat and jsonlint. now i dont us efm anymore. because i dont know wich update cause efm not working. i was tried many roll back plugins and neovim it self but nothing work. i give up now, may be you got lucky with efm.

Reply:
honestly efm are best project if it working. i first imprest by it and your config collection. because it much light weight than other plugin method. i have high hope for this project to be stable and working again. last thanks for the experience it's awsome external hooker/wrapper for formater and linter.

Reply:
Hey man, sorry if efm or this plugin didn't work out for you :( I'd be happy to help fix this with you if you'd like.

If it did work in the older versions, it should be able to work on the newer ones. I think it's just the `lintFormats` that needs fixing.

I will take a look at it later today and probably merge this with another PR for the fix.

I didn't get to mention this before but thanks for contributing though not many people get that motivation to make PRs :)